vsltranscore.py

```diff
@@ -891,6 +891,9 @@ def execute(self):
             ret = self.vap.Dispatch(self.vapCallBack)
             if 0 == ret:
                 time.sleep(0.1)
+            if ret < -3:
+                print(self.vap.error)
+                exit(ret)
```

Comment:
negative numbers are not proper exit codes in *nix systems

Reply:
Right.